src/sentry/static/sentry/app/views/settings/project/projectUserFeedback.jsx

```diff
@@ -0,0 +1,186 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {t} from '../../../locale';
+import AsyncView from '../../asyncView';
+import Button from '../../../components/buttons/button';
+import Form from '../components/forms/form';
+import JsonForm from '../components/forms/jsonForm';
+import Panel from '../components/panel';
+import PanelBody from '../components/panelBody';
+import PanelHeader from '../components/panelHeader';
+import SettingsPageHeader from '../components/settingsPageHeader';
+import TextBlock from '../components/text/textBlock';
+import formGroups from '../../../data/forms/userFeedback';
+
+const CodeBlock = styled.pre`
+  word-break: break-all;
+  white-space: pre-wrap;
+`;
+
+const TextBlockNoMargin = styled(TextBlock)`
+  margin-bottom: 0;
+`;
+
+class ProjectUserReportSettings extends AsyncView {
+  static propTypes = {
+    setProjectNavSection: PropTypes.func,
+  };
+
+  componentWillMount() {
+    super.componentWillMount();
+    this.props.setProjectNavSection('settings');
+  }
+
+  componentDidMount() {
+    window.sentryEmbedCallback = function(embed) {
+      // Mock the embed's submit xhr to always be successful
+      // NOTE: this will not have errors if the form is empty
+      embed.submit = function(body) {
+        this._submitInProgress = true;
+        setTimeout(
+          function() {
+            this._submitInProgress = false;
+            this.onSuccess();
+          }.bind(this),
+          500
+        );
+      };
+    };
+  }
+
+  componentWillUnmount() {
+    window.sentryEmbedCallback = null;
+  }
+
+  getEndpoints() {
+    let {orgId, projectId} = this.props.params;
+    return [
+      ['keyList', `/projects/${orgId}/${projectId}/keys/`],
+      ['project', `/projects/${orgId}/${projectId}/`],
+    ];
+  }
+
+  getInstructions() {
+    let dsn = this.state.keyList.length
+      ? this.state.keyList[0].dsn.public
+      : 'https://public@sentry.example.com/1';
+
+    return (
+      '<!-- Sentry JS SDK 2.1.+ required -->\n' +
+      '<script src="https://cdn.ravenjs.com/2.1.0/raven.min.js"></script>\n\n' +
```

Comment:
Not necessary to change here, but we should actually sub in the _latest_ version of raven.js here instead of hard-pinning to 2.1.0 (which is really old).